internal/database/models/models.go

```diff
@@ -33,3 +33,10 @@ type Story struct {
 	AuthorCountry   string  `json:"author_country"`
 	Author          Author  `gorm:"foreignKey:AuthorFirstName,AuthorLastName,AuthorCountry;references:FirstName,LastName,OriginCountry" json:"author"`
 }
+
+type Tag struct {
+	gorm.Model
+	Name    string `gorm:"not null" json:"name"`
```

Comment:
You should specify the primary key of this table as (Name, StoryID). This will prevent duplicate story tag pairs